lib/datasource/helm/index.ts

```diff
@@ -18,27 +22,27 @@ export async function getPkgReleases({ lookupName, registryUrls }) {
     logger.warn(`Couldn't get index.yaml file from ${helmRepository}`);
     return null;
   }
-  const releases = repositoryData[lookupName];
+  const releases = repositoryData.find(chart => chart.name === lookupName);
   if (!releases) {
     logger.warn(
       { dependency: lookupName },
       `Entry ${lookupName} doesn't exist in index.yaml from ${helmRepository}`
     );
     return null;
   }
-  return {
-    releases,
-  };
+  return releases;
 }
 
-export async function getRepositoryData(repository) {
+export async function getRepositoryData(
+  repository: string
+): Promise<ReleaseResult[]> {
```

Comment:
What was the reason for needing to switch the return value from being an object to it being an array?

Reply:
The previous return type was:

```ts
interface RepositoryData {
  [key: string]: {
    version: string;
    homepage: string;
    sources: string[];
    urls: string[]
  }[]
}
```

Compared with `ReleaseResult` that we use everywhere else:

```ts
interface ReleaseResult {
  name: string;
  homepage: string;
  sourceUrl: string;
  releases: {
    version: string;
    ....
  }[]
  ...
}
```

It was just much easier to re-use the interface we use everywhere else, including the calling method, instead of creating a new type specifically for Helm.

Reply:
Thanks, I missed that